.github/workflows/gitlab-runners-ubi8-git-publish.yaml

```diff
@@ -0,0 +1,36 @@
+name: ubi8-git-publish
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - gitlab-runners/ubi8-git/version.json
+jobs:
+  build:
+    env:
+      context: gitlab-runners/ubi8-git
+      image_name: gitlab-runners-ubi8-git
+    # Only build on redhat-cop repo, do not attempt to build on forks
+    if: github.repository == 'redhat-cop/containers-quickstarts'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get image tags
+        id: image_tags
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          VERSION=$(jq -r '.version' ${context}/version.json)
+          if [ "${VERSION}" == "latest"]; then
+            echo latest
+          else
+            echo "latest,${VERSION}"
+          fi
+      - name: Build and publish image to Quay
+        uses: docker/build-push-action@v1
+        with:
+          path: ${{ env.context }}
+          registry: quay.io
+          repository: redhat-cop/${{ env.image_name }}
```

Comment:
why not make the `redhat-cop` part a variable so that forks can test pushing to their own private repos?

Reply:
Agreed